force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -56,9 +64,17 @@ public with sharing class FlowControlLoopsCollections {
      */
     public static Integer findMax(Integer num1, Integer num2) {
         // Initialize largestNum as null
-        Integer largestNum = null;
+        Integer largestNum;
+
+        if ( num1 == null || num2 == null){
+            largestNum = null;
+        } else if ( num1 > num2) { 
+            largestNum = num1;
+        } else {
+            largestNum = num2;
+        }
 
-        return null; // Replace null with the variable you used to store the result
+        return largestNum; // Replace null with the variable you used to store the result
```

Comment:
Good work. Take a look at the Math.max() method you can use it to save some code. 

https://developer.salesforce.com/docs/atlas.en-us.apexref.meta/apexref/apex_methods_system_math.htm